### PR TITLE
Fix dev/seed.

### PR DESCRIPTION
**Pull request by ndmckinley**
### Description
Doing version truncation causes some issues, and the Version table doesn't have foreign key references, so deletes don't cascade.

### Checklist
- [x] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### Issue(s)

### CODE REVIEW

File: universal-application-tool-0.0.1/app/controllers/dev/DatabaseSeedController.java [hunk 1]
```diff
@@ -253,6 +257,8 @@ private ProgramDefinition insertProgramWithBlocks(String name) {
   }
 
   private void truncateTables() {
+    versionRepository.getActiveVersion().empty(this);
+    versionRepository.getDraftVersion().empty(this);
```
Comment (hmhsu) [inline universal-application-tool-0.0.1/app/controllers/dev/DatabaseSeedController.java:261]: Suggestion (blocking): I'd lean toward truncating version table as well as all the old version would not work, i.e. cannot be revive, after all programs and questions being deleted.

Comment (ndmckinley) [reply universal-application-tool-0.0.1/app/controllers/dev/DatabaseSeedController.java:261]: I'll give that a try.

File: universal-application-tool-0.0.1/app/controllers/dev/DevController.java [hunk 2]
```diff
@@ -19,7 +19,7 @@ public DevController(Environment environment, Config configuration) {
   }
 
   public boolean isDevEnvironment() {
-    if (environment.isDev()) {
+    if (environment.isDev() || environment.isTest()) {
```
Comment (hmhsu) [inline universal-application-tool-0.0.1/app/controllers/dev/DevController.java:22]: Thoughts: I think we might not need this, the only test that touches this set its environment to dev already.

Comment (ndmckinley) [reply universal-application-tool-0.0.1/app/controllers/dev/DevController.java:22]: Yeah, those tests are currently giving me trouble ... suppose I could try another tack.

**Review by hmhsu — APPROVED** [review]
Looks good. Hope the tests pass!

**Pull request merged**